src/components/ButtonsList.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button } from "./Button";
+
+type ButtonsListProps = {
+  filterHandler: (action: string) => void;
+  filter: string;
+  isReversed: boolean;
+};
+
+export const ButtonsList: React.FC<ButtonsListProps> = ({
+  filterHandler,
+  filter,
+  isReversed,
+}) => {
+  return (
+    <div className="buttons">
+      <Button
+        name="alphabet"
+        content="Sort alphabetically"
+        filterHandler={filterHandler}
+        filter={filter}
+        isReversed={isReversed}
+      />
```

Comment:
Remember to mark the active button to explain to the user what action is active. You could conditionally apply a class to the button based on the current `filter` and `isReversed` props to visually indicate which sorting is currently applied.